js/vendor/mini-html5-framework/templater.js

```diff
@@ -0,0 +1,31 @@
+function render(template, context) {
+
+    var found = {},
+        re = /{{\s*([^}]+)\s*}}/g,
+        parsedArray = [],
+        curMatch;
+
+    while(curMatch = re.exec(template)) {
```

Comment:
Ah, funny, I didn't know about this repeated `exec` feature for global regexp's. Perhaps add a comment here to make the use of this feature clear?